homeassistant/components/point/binary_sensor.py

```diff
@@ -71,14 +71,13 @@ def __init__(self, point_client, device_id, device_name):
         super().__init__(
             point_client,
             device_id,
-            DEVICES[device_name].get("device_class"),
+            DEVICES[device_name].get("device_class", device_name),
         )
```

Comment:
Why is this platform changed?

Reply:
It is not but the `device_name` is used in the parent class to set `self_attr_name`:

https://github.com/home-assistant/core/blob/8d2bc1f1092bbb06080c8dce592e2d5d27f30d4d/homeassistant/components/point/__init__.py#L280

Reply:
Please move changes that are not required for the version bump to a separate PR.